Sources/SwiftGodotMacroLibrary/MacroGodot.swift

```diff
@@ -265,7 +269,7 @@ public struct GodotMacro: MemberMacro {
                     return stringName
                 }
                 
-                var implementedOverridesDecl = "override class func implementedOverrides() -> [StringName] {\nsuper.implementedOverrides() + [\n"
+                var implementedOverridesDecl = "override open class func implementedOverrides() -> [StringName] {\nsuper.implementedOverrides() + [\n"
```

Comment:
This is a bug fix for implementedOverrides; this method needs to be declared open or else classes that import it are not able to override.

Reply:
oops, yes